deployment/ccip/changeset/testhelpers/test_helpers.go

```diff
@@ -798,6 +800,128 @@ func DeployTransferableToken(
 	return srcToken, srcPool, dstToken, dstPool, nil
 }
 
+// assuming one out of the src and dst is solana and the other is evm
+func DeployTransferableTokenSolana(
+	t *testing.T,
+	lggr logger.Logger,
+	e deployment.Environment,
+	evmChainSel, solChainSel uint64,
```

Comment:
Do you wanna verify that evmChainSel is EVM family and solChainSel is SVM chain family?